core/operation/src/test/java/uk/gov/gchq/gaffer/operation/export/set/GetSetExportTest.java

```diff
@@ -45,7 +45,7 @@ public void shouldJSONSerialiseAndDeserialise() throws SerialisationException {
         // Then
         assertEquals("key", deserialisedOp.getKey());
         assertEquals("jobId", deserialisedOp.getJobId());
-        assertEquals(0, deserialisedOp.getStart());
+        assertEquals(0, (int) deserialisedOp.getStart());
```

Comment:
If deserialisedOp.getStart() is null you will end up with a null pointer exception here instead of a nice error message making it more difficult to debug.

Reply:
I think this is a merge issue - fixed anyway.